docs/datastructures/overview.md

```diff
@@ -14,9 +14,12 @@ to implement any high level functionality themselves.
 
 ## Eager Loading
 
-When executing a contract, all the fields of the `#[ink(storage)]` struct will be pulled
+When executing a contract, all* the fields of the `#[ink(storage)]` struct will be pulled
 from storage, regardless of whether or not they are used during the message execution.
 
+Note*: This does **not** apply to `Mapping` fields, though, as key lookups in mappings
+are done directly from contract storage.
```

Comment:
```suggestion
:::note

This does **not** apply to `Mapping` fields, though, as key lookups in mappings
are done directly from contract storage.

:::
```

And can you move this to the bottom of the page?

Reply:
Yes, I've moved it as a note at the bottom, i.e.:
> **Note:**

Thanks!